app/backend/src/clj/teet/thk/thk_mapping.clj

```diff
@@ -150,9 +150,7 @@
    "activity_thkupdstamp"
    "activity_teetupdstamp"
    "activity_teetdelstamp"
-   "activity_cost"
-   "activity_procurementno"
```

Comment:
does this mean that these columns are removed from the teet-thk export csv?

Reply:
not necessary, it only means we are not using information from them